itest/list_on_test.go

```diff
@@ -535,4 +535,8 @@ var allTestCases = []*lntest.TestCase{
 		Name:     "custom features",
 		TestFunc: testCustomFeatures,
 	},
+	{
+		Name:     "channel memo during creation",
```

Comment:
Could this perhaps be folded in with an existing test? Just to reduce the number of test cases to run, as each of them have some overhead (and we're well into the full hour or more runtime on those now).

Reply:
I suppose I could stuff this into `testBasicChannelCreationAndUpdates` here:
https://github.com/lightningnetwork/lnd/blob/fd9adaf6ceb3649c07bbb4982bd60dd632e8cda0/itest/lnd_open_channel_test.go#L379

What do you think?

Reply:
Sounds good, yes!